content/efcore/getting-started/manipulating-entities.md

```diff
@@ -1,6 +1,6 @@
 # Manipulating Entities 
  
-Four operations that we often perform on a record in a database are create, read, update, and delete (commonly referred to as "CRUD" operations). We walked through how to read records from a database in the [Querying Data](querying.md) tutorial. Now let's learn how to perform the other CRUD operations which allow us to manipulate records in the database. 
+Four operations that we often perform on a record in a database are create, read, update, and delete (commonly referred to as "CRUD" operations). We walked through how to read records from a database in the [Querying Data](querying.md) tutorial. In this lesson, we'll learn how to perform the other CRUD operations which allow us to manipulate records in the database. 
```

Comment:
~~"in the [Querying Data](querying.md) tutorial."~~ => "in the [Querying Data](querying.md) lesson."